docs/development/docs-site.md

```diff
@@ -0,0 +1,6 @@
+# Docs site
+
+The [Renovate docs site](https://docs.renovatebot.com) is built from [a dedicated publishing repository](https://github.com/renovatebot/renovatebot.github.io) that pulls the source files from [this repository](../usage/).
+
+The publishing process is triggered manually as needed.
```

Comment:
it's normally automatically updated by renovate (git-submodule)

Reply:
Ah, now I understand, thanks. Adapted!